aws/resource_aws_api_gateway_base_path_mapping.go

```diff
@@ -27,17 +28,17 @@ func resourceAwsApiGatewayBasePathMapping() *schema.Resource {
 			"api_id": {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
+				ForceNew: false,
```

Comment:
Nit: Setting the default `false` value for `ForceNew` is extraneous in schemas. 👍 

```suggestion
```